docs/doc.md

```diff
@@ -1,6 +1,6 @@
 # Structure of system dictionary
 
-This is the output of `parse_tpr` and `parse_top`:
+This is the structure of a system dictionary (e.g.: `parse_tpr` and `parse_top`):
 
 ``` 
 system = {
```

Comment:
This is still the dictionary structure, isn't it?

Reply:
yes, just made it clearer that it's not just the output of those functions, but also the structure of the dictionary in the system class